README.md

```diff
@@ -1,12 +1,8 @@
-# Exploring_Hacker_News_Posts
+# Analyzing Hacker News Posts
 In this project, we will be working with a dataset containing information about roughly 20,000 Hacker News Listings. As we explore the dataset, we will also form some initial analysis of our data. With the results of our analysis, we will address the two questions presented at the beginning of the project:
   - Do "Ask HN" or "Show HN" receive more comments on average?
   - Do posts created at a certain time receive more comments on average?
 
-<br/>
+[![Analyzing Hacker News Posts](hn-analysis.png)](https://colab.research.google.com/drive/16C6ijfSvTfi9mliPYq4FqNKXjZ5W2MvX)
 
-**Note:**
-For those who are not familiar with Jupyter Notebook, here are some steps on how to run the code within the notebook:
-  - Look for the toolbar above the code where you can see the dropdowns: 'File' 'Edit' 'View' 'Run' 'Kernel' 'Settings' 'Help'
-  - Select the 'Run' dropdown, and then select the 'Run All Cells' option
-  - OR If you have selected a certain cell you would like to examine, by clicking on that cell, then in the 'Run' dropdown, select the 'Run Selected Cell' option
```

Comment:
I just deleted these instructions since you have the Colab version now. I think if someone wants to run it locally, they'll know how, and if not, they can look at it on Colab.